### PR TITLE
Feat/toast UI integration

### PR DESCRIPTION
**Pull request by DeRaowl**
### What is the change?

I have created a route Events where ToastUI calendar API integration been done. As Remix doesn't support CSS module import from node_modules I have added 3 link's in root.tsx which imports css from CDN link
image

### Is it bug? No

- Steps to repro
- Expected - ToastUI intergration
- Actual - ToastUI intergration

### \*Dev Tested?

- [X] Yes
- [ ] No

#### Platforms

- [C] Web

#### Browsers

- [X] Chrome
- [ ] Safari
- [ ] Firefox

### Before / After Change Screenshots

Before:
It is a new route created hence page is blank

After:
image

> For visual or interaction changes. Can be video / screenshot.
Refer After screenshot

### CODE REVIEW

**Review by harshith-venkatesh — APPROVED** [review]

File: app/components/Calendar.client.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import TUICalendar from '@toast-ui/react-calendar';
+
+function Calendar() {
+  return (
+    <div>
+      <TUICalendar height="100vh" />
+    </div>
+  );
+}
```
Comment (sumitd94) [inline app/components/Calendar.client.tsx:9]: NIT: Arrow function and can we get rid of `return` keyword and directly use `()`?

File: app/components/Calendar.client.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import TUICalendar from '@toast-ui/react-calendar';
```
Comment (sumitd94) [inline app/components/Calendar.client.tsx:1]:
```suggestion
import ToastUICalendar from '@toast-ui/react-calendar';
```

Makes it more readable

File: app/routes/Event.tsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import Calendar from '~/components/Calendar.client';
+
+// This will changed - TODO
```
Comment (sumitd94) [inline app/routes/Event.tsx:3]: Changed to what? can we add a small description? or may create a to-do ticket and update the comment with that link? so we have a track for it?

File: app/routes/Event.tsx [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+const Fallback = () => {
+  return <p>Loading IDE...</p>;
+};
```
Comment (sumitd94) [inline app/routes/Event.tsx:6]:
```suggestion
const Fallback = () => (
  <p>Loading IDE...</p>;
);
```

File: app/routes/Event.tsx [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+
+function Event() {
+  return typeof document !== 'undefined' ? <Calendar /> : <Fallback />;
```
Comment (sumitd94) [inline app/routes/Event.tsx:9]: NIT: can get rid of the `return` keyword

File: app/routes/Event.tsx (hunk 4 above)
Comment (DeRaowl) [reply app/routes/Event.tsx]: Removed return keyword

File: app/routes/Event.tsx (hunk 5 above)
Comment (DeRaowl) [reply app/routes/Event.tsx]: Removed return keyword

File: app/routes/Event.tsx [hunk 6]
```diff
@@ -0,0 +1,8 @@
+import Calendar from '~/components/Calendar.client';
+
+//TODO - create ticket to show spinner as a fallback component(Ticket will be attached)
```
Comment (DeRaowl) [inline app/routes/Event.tsx:3]: @harshith-venkatesh can we create ticket for this? A simple spinner can be shown here.

**Review by harshith-venkatesh — APPROVED** [review]

File: app/routes/Event.tsx (hunk 6 above)
Comment (harshith-venkatesh) [reply app/routes/Event.tsx:3]: Sure, I will go ahead and raise a ticket.
You could go ahead and remove the comment please

**Pull request merged**